src/dashboard/cd-dashboard-projects.vue

```diff
@@ -41,17 +41,28 @@
       userLocale() {
         const langCookie = LocaleService.getUserLocale();
         if (langCookie) {
-          return langCookie.replace(/"/g, '').replace('_', '-');
+          let formattedLocale = langCookie.replace(/"/g, '').replace('_', '-');
+          if (/en-.*/.test(formattedLocale)) {
+            formattedLocale = 'en';
+          }
+          return formattedLocale;
         }
         return 'en';
       },
     },
     methods: {
       async loadProjects(locale = 'en') {
-        const projects = (await ProjectsService.list(locale, { order: 'desc' })).body;
-        if (projects) {
-          this.projects = projects.data.slice(0, 3);
-        }
+        await ProjectsService.list(locale, { order: 'desc' })
```

Comment:
You should be able to just wrap the existing code in try/catch. It works around async/await.

```javascript
try {
  const projects = (await ProjectsService.list(locale, { order: 'desc' })).body;
  if (projects) {
    this.projects = projects.data.slice(0, 3);
  }
} catch() {}
```

Makes it a little cleaner and easier to see what's going on.